Blizzard.IO.RabbitMQ/NetqRabbitConsumer.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using Blizzard.IO.Core;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public class NetqRabbitConsumer<TData> : IConsumer<TData>, IConsumerWithMetadata<TData, RabbitMessageProperties>
+    {
+        private readonly IBus _netqBus;
+        private readonly IDeserializer<TData> _deserializer;
+        private readonly IConcreteTypeDeserializer<TData> _concreteTypeDeserializer;
+        private readonly IQueue _sourceQueue;
+        private readonly IConverter<MessageProperties, RabbitMessageProperties> _coverter;
+        private readonly ILogger<NetqRabbitConsumer<TData>> _logger;
+        
+        private IDisposable _consumeTask;
```

Comment:
This not a Task. this name is confusing cause Im pretty `Task` is not `IDisposable` either. Maybe call it consumeHandler?
(In c and c++ we usually call those things handlers cause they are a reference that to a resource that identify this resource and allow you pass him to other places to do operations on him).

Reply:
f